src/components/dashboard/Claim.js

```diff
@@ -281,8 +294,8 @@ const Claim = props => {
   const handleFaceVerification = () => {
     //if user is not in whitelist and we do not do faceverification then this is an error
     if (Config.zoomLicenseKey == null) {
+      log.error('handleFaceVerification failed', '', new Error('Zoom licensekey missing'), { dialogShown: true })
```

Comment:
Zoom returns corresponding code, it would be logged like Unrecoverable error.
So this check isn't needed here

Reply:
Everything else is fine. Just fix this or explain please why it's needed - and i'll approve the PR

Reply:
This log was written before I made changes here. So I could delete it if needed.

Reply:
yes, delete it please but check if ErrorScreen logs an exception

Reply:
created separate PR for this issue https://github.com/GoodDollar/GoodDAPP/pull/2106
@yaroslav-fedyshyn-nordwhale please check it & review